lib/AST/ProtocolConformance.cpp

```diff
@@ -1367,3 +1368,46 @@ ProtocolConformanceRef::getCanonicalConformanceRef() const {
     return *this;
   return ProtocolConformanceRef(getConcrete()->getCanonicalConformance());
 }
+
+// See swift/Basic/Statistic.h for declaration: this enables tracing
+// ProtocolConformances, is defined here to avoid too much layering violation /
+// circular linkage dependency.
+
+struct ProtocolConformanceTraceFormatter
+    : public UnifiedStatsReporter::TraceFormatter {
+  void traceName(const void *Entity, raw_ostream &OS) const {
+    if (!Entity)
+      return;
+    const ProtocolConformance *C =
+        static_cast<const ProtocolConformance *>(Entity);
+    OS << "<conformance ";
```

Comment:
Conformances already have a `printName` method that probably covers this.

Reply:
Ok, fixed.